catalog-rest-service/src/main/java/org/openmetadata/catalog/jdbi3/EntityRepository.java

```diff
@@ -328,13 +422,19 @@ public EntityUpdater(T original, T updated, boolean patchOperation) {
       this.patchOperation = patchOperation;
     }
 
-    public final void update() throws IOException {
+    /**
+     * Compare original and updated entities and perform updates. Update the entity version and track changes.
+     */
+    public final void update() throws IOException, ParseException {
       updated.setId(original.getId());
       updateDescription();
       updateDisplayName();
       updateOwner();
       updateTags(updated.getFullyQualifiedName(), "tags", original.getTags(), updated.getTags());
       entitySpecificUpdate();
+
+      // Store the updated entity
+      store();
```

Comment:
May this call be `storeEntity`?

Reply:
This an internal method of `EntityUpdater` diff from `EntityRepository`. We can change the name, if necessary.

Reply:
What would you think of `storeUpdates`?

Reply:
Good idea. I will change it to `storeUpdate`.